docs/source/pre_application/cloud_provider_tutorial.rst

```diff
@@ -0,0 +1,140 @@
+.. _cloud-provider-tutorial:
+
+=============================================================================
+Tutorial for Creating a Threshold PRE Testnet Node on a Remote Cloud Provider
+=============================================================================
```

Comment:
A bit busy in the sidebar:

![Screen Shot 2022-03-30 at 3 29 05 PM](https://user-images.githubusercontent.com/19150641/160915534-193acb64-e78e-47b3-aa7d-4b23b9493ee2.png)

```suggestion
=================================
PRE Testnet Node Cloud Automation
=================================
```